Makefile

```diff
@@ -331,6 +331,16 @@ test-docker-clean: ## Clean up Docker environment used in test-docker target
 
 ##@ Tooling
 
+.PHONY: diagrams
+diagrams:
+	archview -skip-class "Peer,Master Database" -trim-prefix storj.io/storj/satellite/   ./satellite/...   | dot -T svg -o satellite.svg
+	archview -skip-class "Peer,Master Database" -trim-prefix storj.io/storj/storagenode/ ./storagenode/... | dot -T svg -o storage-node.svg
+
+.PHONY: diagrams-graphml
+diagrams-graphml:
```

Comment:
the `diagrams` command works well for me, but this one seems to stack everything on top of each other when I open the files with a `graphml` editor. The connections on the graph seem to be in place, but it doesn't look nice visually. Is this intended?

Reply:
Yes, that's to be expected. The exported `graphml` doesn't contain any positioning information, you need to run a layouting algorithm in the editor. In yEd it's `Layout -> Hierarchical`